README.md

```diff
@@ -63,21 +63,20 @@ _* Not verified. Anyone who is able to verify that this driver works or not is e
 #### Prerequisites
 
 * Internet connection
-* [ROS](http://wiki.ros.org/ROS/Installation) _Indigo_ or _Jade_
-* Ubuntu packages: `python-rosdep`, `python-catkin-tools`
+* [ROS](http://wiki.ros.org/ROS/Installation) _Foxy_
+* Ubuntu packages: `python3-rosdep2`, `python3-colcon-common-extensions`
```

Comment:
I think `python3-rosdep` should continue to work. Is there an issue?

Reply:
No issue. I followed Ubuntu's suggestion of installing `python3-rosdep2`, but from reading the docs I think `python3-rosdep` is indeed the correct one.